CompilerSource/compiler/event_reader/event_parser.cpp

```diff
@@ -22,6 +22,7 @@
 #include <vector>
 #include <map>
 using namespace std;
+#define tostring to_string
```

Comment:
is there really a need for this? why not just atually replace it...

Reply:
More useless line changes.

Reply:
hardly, we have toString, to_string, tostring, and std::to_string in  our sources now some are the same func implemented multiple times, some are crappy macros like this and some are usally useful functions. Adding another macro only piles onto that mess and makes it harder to find the actual function being called.